packages/@aws-cdk/aws-cognito/lib/user-pool.ts

```diff
@@ -34,6 +34,13 @@ export interface SignInAliases {
    * @default false
    */
   readonly preferredUsername?: boolean;
+
+  /**
+   * Whether sign-in aliases should be evaluated with case sensitivity.
+   * For example, when this option is set to false, users will be able to sign in using either `MyUsername` or `myusername`.
+   * @default true
+   */
+  readonly caseSensitive?: boolean;
```

Comment:
Move this out of the `SignInAliases` struct, into the top level struct - `UserPoolProps` - and call it `signInCaseSensitive`.